### PR TITLE
setup AdoptionChart for translation

### PR DESCRIPTION
**Pull request by corwintines**
## Description
- Setup AdoptionChart on [what-is-ethereum](https://ethereum.org/en/what-is-ethereum) for translation

### CODE REVIEW

File: src/intl/en/page-what-is-ethereum.json [hunk 1]
```diff
@@ -96,5 +96,16 @@
   "page-what-is-ethereum-the-year-in-ethereum-2021-desc": "Jan 17, 2022 - Josh Stark and Evan Van Ness",
   "page-what-is-ethereum-atoms-institutions-blockchains": "Atoms, Institutions, Blockchains",
   "page-what-is-ethereum-atoms-institutions-blockchains-desc": "- Why blockchains matter?",
-  "page-what-is-ethereum-ethereum-in-numbers-title": "Ethereum in numbers"
+  "page-what-is-ethereum-ethereum-in-numbers-title": "Ethereum in numbers",
+  "adoption-chart-column-2010-label": "2010",
```
Comment (minimalsm) [inline src/intl/en/page-what-is-ethereum.json:100]: Do we need to translate years? cc: @lukassim

Comment (minimalsm) [reply src/intl/en/page-what-is-ethereum.json:100]: Context:

Screenshot 2023-01-06 at 09 59 27

Comment (lukassim) [reply src/intl/en/page-what-is-ethereum.json:100]: I think we can leave the years untranslated.
Having quickly looked through the top 5 non-Latin languages in Crowdin, all of them use Latin numbers for dates.

**Comment by corwintines** [comment]
@minimalsm @lukassim fixed

**Comment by minimalsm** [comment]
Build failed, rerunning

File: src/intl/en/page-what-is-ethereum.json (hunk 1 above)
Comment (wackerow) [reply src/intl/en/page-what-is-ethereum.json]: I would use the Intl NumberFormat function to style numbers according to locale
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat
They shouldn't need to be translated.

**Review by wackerow — APPROVED** [review]

**Pull request merged**